docker/nuke.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
```

Comment:
One last thing, did you want to add the same trick we have in other bash files so this can be run from `./docker/nuke.sh`? I think without this it won't work unless you cd into the docker dir manually.

```
DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
cd $DIR && docker-compose -p datahub down -v
cd $DIR && docker-compose rm -f -v
```

Reply:
SGTM